### ISSUE TITLE
No task end if folder is empty

### ISSUE THREAD

**Issue by andrey-hohlov** [reducejs/gulp.spritesmith-multi#2]
Hi!

I have this file structure:

```
├── sprites
│   ├── sprite-1
│   │   └── image1.png
│   │   └── image2.png 
│   ├── sprite-2
│   │   └── image1.png
│   │   └── image2.png 
```

If there is no files in both folders, like a

```
├── sprites
│   ├── sprite-1
│   │   └── empty
│   ├── sprite-2
│   │   └── empty
```

Task not ended:

```
[14:37:09] Starting 'sprite'...
Process finished with exit code 0
```

It's breaks runSequence for example.

**Comment by zoubin** [comment]
@andrey-hohlov Thank you for reporting this!
Did you happen to continue the pipeline as https://github.com/zoubin/gulp.spritesmith-multi#continuing-the-pipeline? There is a bug in cases where no images found. I'm going to fix it today.

However, if you use it like https://github.com/zoubin/gulp.spritesmith-multi#example, empty directories should not cause the gulp task pending.

**Comment by andrey-hohlov** [comment]
@zoubin my task like as your example: https://github.com/zoubin/gulp.spritesmith-multi#continuing-the-pipeline because i need save images and styles into differet directories.

Waiting for you fix, it's really good module for me)

**Comment by zoubin** [comment]
@andrey-hohlov The fix is published in the release v2.2.1. If anything goes wrong, please tell me.

**Comment by andrey-hohlov** [comment]
It works, thank you!
